src/components/metadata/FindingaidesMetadata.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
Could we rename this component to FindingaidsMetadata? (no e in findingaids)?

Reply:
Thanks for catching that!